components/brave_wallet_ui/components/shared/authorize-hardware-device/authorize-hardware-device.tsx

```diff
@@ -32,7 +32,7 @@ export const AuthorizeHardwareDeviceIFrame = (props: Props) => {
   const src =
     LEDGER_BRIDGE_URL +
     `?targetUrl=${encodeURIComponent(window.origin)}` +
-    `&bridgeType=${bridgeTypeFromCoin(props.coinType)}`
+    `&bridgeType=${encodeURIComponent(bridgeTypeFromCoin(props.coinType))}`
```

Comment:
This looks fine to me - that said `bridgeTypeFromCoin` seems to throw an error for unknown coin types, so its not strictly necessary - probably good practise though